workflows/prognostic_c48_run/runfv3

```diff
@@ -42,18 +42,7 @@ function createRun {
     fv3config="$2"
     runfile="$3"
 
-
-    if [[ $# != 3 ]]
-    then 
-        usage
-        exit 1
-    fi
-
-    verifyUrlEmpty "$url"
-
-    echo "Setting up iterations"
-    gsutil cp "$fv3config" "$url/fv3config.yml"
-    gsutil cp "$runfile" "$url/runfile.py"
+    create_run $url $fv3config $runfile
```

Comment:
I suggest prefixing the command name with  `segmented-run` since "run" is overloaded, and I envision the segmented run actually being a python class with methods at some point. Feel free to change to underscore, but I think it's uglier and required shift when typing, so is harder on my pinkies.

```suggestion
    segmented-run-create $url $fv3config $runfile
```